UC-Berkeley-Extension/global
# Info Sessions

## Asana Ticket(s)
- See ticket [Angular2 Templates](https://app.asana.com/0/1174970560648905/1197472455447401).

## Description
• Create Info Sessions Angular template that matches campus styling. 
• Create [Info Sessions on the Styleguide](http://localhost:3002/?p=molecules-info-sessions-list)
To replace this component from the home page
![image](https://user-images.githubusercontent.com/17348237/95836564-71755300-0cf4-11eb-9808-9a77b6d1fd55.png)

• Also in this pull request is [adding Voices List](http://localhost:3002/?p=molecules-voices-list) to the Styleguide

## Testing instructions

* Checkout this branch
* Run `../scripts/setup.sh` to re-install Drupal
* Go to https://global.ddev.site/ 
 * Observe the `Angular: Info Sessions List ` block is displayed on the homepage and has content.
* Login using `drush uli`
* Go to [Blocks](https://global.ddev.site/admin/structure/block)
* Observe that the `Angular: Info Sessions List` block is enabled in the Content section.
* [Edit the block](https://global.ddev.site/admin/structure/block/manage/angularinfosessionslist?destination=/admin/structure/block)
* Observe that you can edit the following values:
  - Number of Sessions
  - areas
  - programs
* Set areas to `x` and save the block
* Go to the homepage
* Observe that the block only shows sessions from x

## Remaining tasks
- [x] Make Number of Sessions field limit the number of sessions to show
- [x] Change functionality to remove ability to specify areas (it's all International)
- [x] Move styling to Styleguide
- [x] Add Info Sessions to default build
